src/Map.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react';
+import ReactMapGL from 'react-map-gl';
+import 'mapbox-gl/dist/mapbox-gl.css';
+
+interface MapProps {
+  latitude: number,
+  longitude: number
+}
+
+const Map: React.FC<MapProps> =  ({latitude, longitude}) =>{
```

Comment:
```suggestion
const Map =  ({latitude, longitude}: MapProps) =>{
```

Reply:
Oh, is that how I'm supposed to do it? Wow, TS is crazy